flutter/flutter-macos-demo-project/codemagic.yaml

```diff
@@ -22,8 +22,10 @@ workflows:
             --create
       - name: Fetch Mac Installer Distribution certificates
         script: |
+          # HINT: We can use MAC_APP_DISTRIBUTION if looking to package the artifact before submitting it to App Store. 
+          # And please note that the certificate type must match the one we are using in the Package macOS application script.
```

Comment:
perhaps let's say "in the Package macOS application script down below", otherwise it is hard to guess what script you are referring to.